api/lib/application/sessions/session-controller.js

```diff
@@ -81,14 +81,20 @@ const getAttendanceSheet = async function (request, h, dependencies = { tokenSer
     .header('Content-Disposition', `attachment; filename=${fileName}`);
 };
 
-const getSupervisorKitPdf = async function (request, h, dependencies = { tokenService, supervisorKitPdf }) {
+const getSupervisorKitPdf = async function (
+  request,
+  h,
+  dependencies = { tokenService, requestResponseUtils, supervisorKitPdf }
+) {
   const sessionId = request.params.id;
   const token = request.query.accessToken;
   const userId = dependencies.tokenService.extractUserId(token);
+  const locale = request.query.lang;
   const sessionForSupervisorKit = await usecases.getSupervisorKitSessionInfo({ sessionId, userId });
 
   const { buffer, fileName } = await dependencies.supervisorKitPdf.getSupervisorKitPdfBuffer({
     sessionForSupervisorKit,
+    locale,
```

Comment:
Pour éviter la confusion avec la locale de l'utilisateur, je resterais bien sur lang ici 
```suggestion
  const lang = request.query.lang;
  const sessionForSupervisorKit = await usecases.getSupervisorKitSessionInfo({ sessionId, userId });

  const { buffer, fileName } = await dependencies.supervisorKitPdf.getSupervisorKitPdfBuffer({
    sessionForSupervisorKit,
    lang,
```